src/dynamicTypes/widget.js

```diff
@@ -18,10 +18,19 @@ const getMembersTypes = members =>
     })
     .join("\n");
 
-const getWidgetTypeDeclarations = ({ name, members = {}, events = {} }) => {
+const getWidgetDeclaration = ({ name, members = {}, events = {} }) => {
   return `interface ${name} extends $w.IFrame { 
     ${getMembersTypes(members)} ${getEventHandlersTypes(events)}}`;
 };
+
 module.exports = {
-  getWidgetTypeDeclarations
+  getWidgetDeclaration,
+  getWidgetsTypeDeclarations: function getWidgetsTypeDeclarations(widgets) {
```

Comment:
consider declaring the function above (imo more readable)

Reply:
True, done